smart_card_connector_app/src/window-about-showing.js

```diff
@@ -46,7 +46,7 @@ const openAboutElement =
 function openAboutClickListener(e) {
   e.preventDefault();
 
-  chrome.app.window.create(ABOUT_WINDOW_URL, ABOUT_WINDOW_OPTIONS);
+  chrome.windows.create({url: ABOUT_WINDOW_URL, type: "popup", width: 700, height: 500});
```

Comment:
nit: Let's introduce constants for "700" and "500", so that these magic numbers aren't scattered through the code, but are rather near the top of the file where they're easily discoverable.